### DISCUSSION TITLE
Using .init file a input

### DISCUSSION THREAD

**Discussion by Archie-2021** [lxqt/lxqt#2479] [Q&A]
I am trying to use qterminal.init file to keep my setting and use it as a way to change the settings. But it seems that If I dont use GUI in add in preference to change the setting they would be over written.

Is there a way around this? Am I missing something here?

Thanks

**Comment by stefonarch** [comment]
Where there is a GUI editing of configuration files is not recommended. Monitoring them continuously would be a waste of resources. Closing app, editing and restart it could work but syntax errors could be tricky.

**Reply by tsujan** [reply]
While agreeing with @stefonarch, I add that if you want to take the risk and edit the config file of an app manually, you'll first need to *close all instances of the app* before doing so; otherwise, the edited file might be overwritten by the running app on exiting. This isn't limited to QTerminal.

**Reply by Archie-2021** [reply]
Thanks, I always forget to close this app becuase it is a terminal and I have on instance open all the times.
